influxql/ast.go

```diff
@@ -372,7 +372,7 @@ func (s *CreateUserStatement) String() string {
 	_, _ = buf.WriteString("CREATE USER ")
 	_, _ = buf.WriteString(s.Name)
 	_, _ = buf.WriteString(" WITH PASSWORD ")
-	_, _ = buf.WriteString(s.Password)
```

Comment:
Not sure if this is ok or not.  I just fixed a bug where  the`String()` representation of a statement was not parable.  This might cause similar issues. @dgnorton might know better.